airbyte-integrations/connectors/source-crowdstrike-falcon/setup.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+
+from setuptools import find_packages, setup
+
+MAIN_REQUIREMENTS = [
+    "airbyte-cdk~=0.2",
```

Comment:
Please add pydantic here. Pydantic is installed because its probably installed by airbyte-cdk.